crypto/dynamic_loading_test.c

```diff
@@ -105,13 +105,15 @@ int main(int argc, char *argv[]) {
     exit(1);
   }
 
-  printf("PASS\n");
+  fprintf(stdout, "PASS\n");
+  fflush(stdout);
```

Comment:
What was happening if you didn't have this? Shouldn't it get flushed when main exits? How did you figure this out?

Reply:
The issue was `main` never exits if the tests hang :(. Without `fflush`, `PASS\n` would be buffered and never printed. I configured the go test runner to dump the output if the tests were hanging, so this had been pretty obvious when I was debugging this.
* `fflush` documentation under `EXAMPLES`: https://pubs.opengroup.org/onlinepubs/000095399/functions/fflush.html

Reply:
Oh interesting, so the tests ran and did all the work but hangs on exit.